resources/inputs/form-schemas/auth-schema.ts

```diff
@@ -0,0 +1,8 @@
+import * as yup from 'yup';
+
+const auth_schema = yup.object().shape({
+  email: yup.string().email('Invalid email format').required('Email is required'),
+  password: yup.string().min(8, 'Password must be at least 8 characters').required('Password is required'),
+});
```

Comment:
This can actually go on the component, as is specific for the auth

resources have no module related logic.